src/index.html

```diff
@@ -5,8 +5,454 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="header__logo-moyo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+            href="#"
+            class="header__nav-link is-active"
+            >
```

Comment:
Pay attention on indents - fix all similar cases
```suggestion
            <a
              href="#"
              class="header__nav-link is-active"
            >
```